chain/db/store.go

```diff
@@ -264,12 +264,12 @@ func (cs *ChainStore) persist(b *Block) error {
 			return err
 		}
 
-		if txn.UnsignedTx.Payload.Type != CoinbaseType && txn.UnsignedTx.Payload.Type != CommitType {
-			pg, _ := ToCodeHash(txn.Programs[0].Code)
-			acc := states.GetOrNewAccount(pg)
+		if txn.UnsignedTx.Payload.Type != CommitType {
+			pg, _ := txn.GetProgramHashes()
```

Comment:
It's better to handle error (not just here but everywhere)

Reply:
I will handle all these errors everywhere  in next commit .